nix-install.sh

```diff
@@ -241,39 +214,14 @@ get_architecture() {
     fi
 
     case "$_ostype" in
-
-        Android)
-            _ostype=linux-android
-            ;;
-
         Linux)
             check_proc
-            _ostype=unknown-linux-$_clibtype
+            _ostype=linux
             _bitness=$(get_bitness)
```

Comment:
I think we can delete _bitness, get_bitness, get_endianness, is_host_amd64_elf, and a few nearby variables like _clibtype